resources/idlharness.js

```diff
@@ -3152,11 +3152,25 @@ function idl_test(srcs, deps, idl_setup_func) {
  */
 function fetch_spec(spec) {
     var url = '/interfaces/' + spec + '.idl';
-    return fetch(url).then(function (r) {
+    return fetch(url).then(function(r) {
         if (!r.ok) {
             throw new IdlHarnessError("Error fetching " + url + ".");
         }
         return r.text();
+    }).then(function(idl) {
+        if (spec == 'cssom') {
+            // CSSOMString is an implementation-defined typedef of either
+            // DOMString or USVString in CSSOM:
+            // https://drafts.csswg.org/cssom/#cssomstring-type
+            // Because of this, neither typedef is included in cssom.idl.
+            // We need some definition, and use DOMString because USVString
+            // has additional requirements in type conversion and could result
+            // in spurious failures for implementations that use DOMString.
+            // Note: No newline is added to preserve line numbers.
+            return 'typedef DOMString CSSOMString;' + idl;
+        }
```

Comment:
This is an odd place to put it; can we just always add it in the `IdlArray` constructor?

Reply:
I don't think special-casing it in the testing framework is the right approach. The fact that it's defined somewhere in the bikeshed source (differently than an IDL snippet) should be handled by the scraper, or importer, but not by the idlharness or test layer.